.github/workflows/compare_translations.py

```
@@ -0,0 +1,212 @@
+        translation: Loaded translation
+    """
+    try:
+        with open(filepath, "r", encoding="utf-8") as file:
+            content = file.read()
+            if not content.strip():
+                raise ValueError(f"File {filepath} is empty.")
+            translation = json.loads(content)
+            flattened_translation = flatten_json(translation)
+        return flattened_translation
+    except json.JSONDecodeError as e:
+        raise ValueError(f"Error decoding JSON from file {filepath}: {e}")
```

Ruff (0.7.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)